src/commands/lambda/functions/commons.ts

```diff
@@ -343,11 +343,17 @@ export const getLayerArn = (
   }
   const account = settings?.layerAWSAccount ?? DEFAULT_LAYER_AWS_ACCOUNT
   const isGovCloud = region.startsWith('us-gov')
+  let arnBuilt
   if (isGovCloud) {
-    return `arn:aws-us-gov:lambda:${region}:${GOVCLOUD_LAYER_AWS_ACCOUNT}:layer:${layerName}`
+    arnBuilt = `arn:aws-us-gov:lambda:${region}:${GOVCLOUD_LAYER_AWS_ACCOUNT}:layer:${layerName}`
+  } else {
+    arnBuilt = `arn:aws:lambda:${region}:${account}:layer:${layerName}`
+  }
```

Comment:
I might slightly favor a ternary here:
```suggestion
  let arnBuilt = isGovCloud
    ? `arn:aws-us-gov:lambda:${region}:${GOVCLOUD_LAYER_AWS_ACCOUNT}:layer:${layerName}`
    : `arn:aws:lambda:${region}:${account}:layer:${layerName}`
```